selenium-system-test/src/test/resources/features/confirmDetails/Applicant confirms details.feature

```diff
@@ -0,0 +1,14 @@
+@HTS-371 @HTS-379
+Feature: Applicant confirms details
+
+  Scenario: Applicant confirms details
+    Given an applicant has the following details:
+      |field          |value            |
+      |first name     |FirstName        |
+      |last name      |LastName         |
+      |NINO           |<auto-generated> |
+      |date of birth  |20/12/1980       |
+      |email address  |user@test.com    |
+
+    When an applicant passes the eligibility check
+    Then they see their details
```

Comment:
instead of using the `Given` step to set `vars` in the `UserDetailsStep` class is it not possible to do something like:
```
Scenario: Applicant confirms details
When an applicant passes the eligibility check with the following details:
       |field              |value                    |
       |first name     |FirstName            |
       |last name      |LastName            |
       |NINO           |<auto-generated> |
       |date of birth  |20/12/1980         |
       |email address  |user@test.com    |
  Then they see the following details:
       |field              |value                    |
       |first name     |FirstName            |
       |last name      |LastName            |
       |NINO           |<auto-generated> |
       |date of birth  |20/12/1980         |
       |email address  |user@test.com    |
```

This avoids `vars` floating around which means if we want to re-use these step definitions its clearer what parameters are going to be used